guides/micronaut-rabbitmq/analytics/groovy/src/main/resources/application.yml

```diff
@@ -0,0 +1,18 @@
+micronaut:
+  application:
+    name: analytics
+---
+#tag::port[]
+micronaut:
+  server:
+    port: 8081 # <1>
```

Comment:
I think this should use such a strategy: https://github.com/micronaut-projects/micronaut-guides/pull/353